demos/sjtwo/Servo/source/main.cpp

```diff
@@ -0,0 +1,57 @@
+#include "L1_Peripheral/lpc40xx/pwm.hpp"
+#include "L2_HAL/actuators/servo/servo.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"
+
+int main()
+{
+  LOG_INFO("Servo application starting...");
+
+  // Creating PWM on pin 2.0
+  sjsu::lpc40xx::Pwm p2_0(sjsu::lpc40xx::Pwm::Channel::kPwm0);
+
+  // Create servo class and and give it the PWM p2_0.
+  sjsu::Servo servo2_0(p2_0);
+
+  // Initialize servo class with default 50 Hz frequency.
+  // 50 Hz allows a pulse width from 0 us to 20,000 us.
+  servo2_0.Initialize();
+
+  // When all of the bounds of the servo class are set, the servo class will
+  // map your degrees to microseconds. With the below example, 0 degrees will
+  // represent 500 us, 180 will represent 2500 us, and by linear correlation
+  // 90 degrees will reperesnt 1500 us.
+
+  // Set the pulse wideth bounds to be 500 us and 2500 us.
+  servo2_0.SetPulseBounds(500, 2500);
+  // Set the angle bounds of the servo to be 0 degrees and 180 degrees
+  servo2_0.SetAngleBounds(0, 180);
+
+  while (true)
+  {
+    // Take ~ 60 seconds to go from 0 degrees to 180 desgrees
+    for (float i_pulse_length = 0; i_pulse_length < 180; i_pulse_length++)
+    {
+      servo2_0.WriteAngle(i_pulse_length);
+      sjsu::Delay(167);
+    }
+
+    // Take ~ 60 seconds to go from 180 degrees to 0 degrees by directly
+    //  manipulating the pulse width.
+    for (float i_pulse_length = 2500; i_pulse_length > 500; i_pulse_length--)
+    {
+      servo2_0.WriteMicroseconds(i_pulse_length);
+      sjsu::Delay(15);
+    }
+
+    // Command the servo to go to 180 degrees as fast as the servo will allow.
```

Comment:
Can you make the rotate 180 -> 0 and vise versa the first thing that happens in the demo. It makes it obvious that the demo has started when the user first loads this demo onto their board.